modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/deployers/terraform/config/TerraformBootConfig.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.deployment.deployers.terraform.config;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import lombok.Getter;
+import org.apache.commons.lang3.StringUtils;
+import org.eclipse.xpanse.modules.models.service.deploy.exceptions.TerraformBootRequestFailedException;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+/**
+ * Configuration class of terraform-boot env.
+ */
+@Configuration
+@Profile("terraform-boot")
+public class TerraformBootConfig {
+
+    @Getter
+    @Value("${webhook.hostname}")
+    private String clientHost;
+
+    @Getter
+    @Value("${server.port}")
+    private String clientPort;
+
+    @Getter
+    @Value("${webhook.request.base.uri}")
```

Comment:
base uri and service URIs are defined in the WebhookApi class. We must move this to one single place and read it from both places.